src/main/java/org/team1540/robot2023/commands/vision/TurnToGamePiece.java

```diff
@@ -48,10 +48,19 @@ public TurnToGamePiece(Drivetrain drivetrain, CommandXboxController controller,
         this.angleSupplier = angleSupplier;
         this.gamepiece = gamepiece;
     }
-
+    long time; 
     @Override
     public void initialize() {
-        limelight.setPipeline(Limelight.Pipeline.GAME_PIECE);
+        limelight.setLedState(Limelight.LEDMode.OFF);
+        long start = System.currentTimeMillis();
+        //limelight.setPipeline(Limelight.Pipeline.GAME_PIECE);
+        limelight.setPipelineBad();
+        limelight.setLedState(Limelight.LEDMode.ON);
+        while(limelight.getLedState() == 1){
+            System.out.println("switching pipeline"); 
+        }
+        long end = System.currentTimeMillis(); 
+        time = end - start; 
```

Comment:
Sana what is this

Reply:
I said in the commit to ignore this... ik it's horrible dw